ddclient

```diff
@@ -2183,7 +2156,7 @@ sub get_ip {
     $arg = '' unless $arg;
 
     if ($use eq 'ip') {
-        $ip  = opt('ip', $h);
+        $reply  = opt('ip', $h);
```

Comment:
Values that aren't exactly an IP address should be rejected:

```suggestion
        $ip = opt('ip', $h);
        $ip = undef if !is_ipv4($ip) && !is_ipv6($ip);
```

Reply:
Updated and added a warning message.